CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Minor: The bot now uses the BTTV v3 API, which should fix some cases where the bot considered more emotes to be enabled than were actually supposed to be enabled.
 - Minor: The points gain rate information at the top of the points page is now dynamically updated based upon your settings for the "Chatters Refresh" module.
 - Minor: "dev" config flag is now respected in web, properly omitting any git information in its footer
+- Minor: The subscriber badge is now automatically downloaded on web application startup. Which version of the subscriber badge should be downloaded can be configured in config.ini under the `web` section using the `subscriber_badge_version` key. Setting the `subscriber_badge_version` key to `-1` disables the sub badge downloading, in case you want to use a custom subscriber badge (or an old one that you don't want to overwrite)
```

Comment:
Example value and comment for `subscriber_badge_version` should be added to the example config

Reply:
Ready for re-review